ts/input/tex/colortbl/ColortblConfiguration.ts

```diff
@@ -0,0 +1,150 @@
+import {ArrayItem} from '../base/BaseItems.js';
+import {Configuration, ParserConfiguration, ConfigurationHandler} from '../Configuration.js';
+import {CommandMap} from '../SymbolMap.js';
+import TexParser from '../TexParser.js';
+import TexError from '../TexError.js';
+import {MmlNode} from '../../../core/MmlTree/MmlNode.js';
+
+import {TeX} from '../../tex.js';
+
+export type ColorData = {
+  cell: string;
+  row: string;
+  col: string[];
+};
```

Comment:
Would that not be better as an interface, in case we need to extend it at some point?